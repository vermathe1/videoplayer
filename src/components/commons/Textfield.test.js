import React from 'react'
import renderer from 'react-test-renderer'
import Textfield from './Textfield.js'

describe('<Textfield />', () => {
  it(' matches the snapshop', () => {
    const tree = renderer.create(<Textfield />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
