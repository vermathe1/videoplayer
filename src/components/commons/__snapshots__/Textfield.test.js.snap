// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Textfield />  matches the snapshop 1`] = `
<div>
  <label />
  <input
    onChange={[Function]}
    required={true}
  />
</div>
`;
